### PR TITLE
Fix input padding

### LINKED ISSUE #4182: Padding Project Color looks funky
image

### PR DESCRIPTION
**Pull request by Nitwel**
closes #4182

### CODE REVIEW

File: app/src/components/v-input/v-input.vue [hunk 1]
```diff
@@ -281,7 +281,9 @@ body {
 		flex-grow: 1;
 		align-items: center;
 		height: 100%;
-		padding: 0 var(--input-padding);
+		padding: var(--input-padding);
+		padding-top: 0px;
+		padding-bottom: 0px;
```
Comment (rijkvanzanten) [inline app/src/components/v-input/v-input.vue:286]: How does this fix it? Isn't `padding: 0 var(--input-padding);` the same as

```css
padding-top: 0;
padding-right: var(--input-padding);
padding-bottom: 0;
padding-left: var(--input-padding);
```

Comment (Nitwel) [reply app/src/components/v-input/v-input.vue:286]: Haha, that's what you think until you notice that `--input-padding` is `12px 12px 12px 12px` so adding a 0 before it wont work.

Comment (rijkvanzanten) [reply app/src/components/v-input/v-input.vue:286]: https://media.giphy.com/media/d3mlE7uhX8KFgEmY/giphy.gif

Only within the context of the color interface though! Maybe we should fix it just in there?


CleanShot 2021-02-19 at 13 45 18@2x

Comment (Nitwel) [reply app/src/components/v-input/v-input.vue:286]: I would say how it is is the right way to go, otherwhise we would have to do something like `.v-input .input {padding-left: 4px}` which would defeat the purpose of having `--input-padding` in the first place.

Comment (rijkvanzanten) [reply app/src/components/v-input/v-input.vue:286]: I disagree. The --input-padding sets the default, but only the color interface has to override the padding-left for it's particular use case. We don't use that anywhere else, so i see it as an exception

**Review by rijkvanzanten — APPROVED** [review]

**Pull request merged**